src/main/java/com/kobylynskyi/graphql/codegen/supplier/SchemaFinder.java

```diff
@@ -15,7 +15,7 @@
  */
 public class SchemaFinder {
 
-    public static final String DEFAULT_INCLUDE_PATTERN = ".*\\.graphqls";
+    public static final String DEFAULT_INCLUDE_PATTERN = ".*\\.graphql[s]?";
```

Comment:
This does work, but the `[]` in regex are normally used for character classes ("any of these characters").
Here we're defining a character class containing only one letter, so it's not really necessary, `.*\\.graphqls?` should be sufficient.
That being said, maybe we can use `()` for readability purposes.

Reply:
Changed as per your suggestion.
Thank you, @joffrey-bion !